mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/223 doors open for characters

Closes #223 

Note, this is built off the `issue/226` branch.

This allows the doors to open for the work users, the IT Staff, and the Security Guard.

To verify:

- -Run the Physical Security scenario
- -Hire a couple security staff
- -Scrap the user's computers to get them wandering
- -Hit Play 
- -All the characters should have the doors open for them.
